### DISCUSSION TITLE
How can I create an image link?

### DISCUSSION THREAD

**Discussion by s-ohnishi** [phalcon/cphalcon#16062] [Q&A] [answered]
I'm trying v5.0.0RC4 with PHP8.1.

I wrote in Volt,
```{{ tag.a(url('ctrl/act'),tag.img(url('img/some.png'))) }}```
 expecting `<a href="/ctrl/act"><img src="/img/some.png" /></a>`
But, it becomes `<a href="/ctrl/act">&lt;img src=&quot;/img/some.png&quot; /&gt;</a>`
Since the img tag part is escaped, it defeats the purpose.

```{{ link_to(url('ctrl/act'),tag.img(url('img/some.png'))) }}```
 produces as expected.
Does it mean that TagFactory has not been able to handle it yet?

I heard that Phalcon\Tag is left for backward compatibility, so I would like to describe it as much as possible with TagFactory.

**Comment by niden** [comment, marked as answer]
I have not tried this but can you issue the image as `raw`?

Something like this

```twig
{{ tag.a( url('ctrl/act'), tag.img(url('img/some.png')), [], true  ) }}
```

So the `a` should not escape the HTML that the `img` generates.

**Reply by s-ohnishi** [reply]
Thank you very much.
The target HTML was generated without any problems.
I should have seen the .zep file, but I didn't understand it.
It was very helpful.

**Comment by niden** [comment]
I am working my way through the documents, I haven't gotten to `H` for `Html` to offer examples etc. Soon though, most likely this week, and then you can check the docs for stuff like that :)

**Reply by s-ohnishi** [reply]
I know you write manuals.
This is because Crowdin frequently sends me an email saying "new string to translate".

I know you're busy, but it would be great if you could remember that you have a few questions left here and let us know once in a while.

**Reply by niden** [reply]
Yes I know. I will get to those shortly.
